specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBManagedCassandraDataCenterGetInstanceView.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "cassandra-prod-rg",
+    "api-version": "2020-06-01-preview",
+    "clusterName": "cassandra-prod",
+    "dataCenterName": "dc1"
+  },
+  "responses": {
+    "202": {},
+    "200": {
+      "body": {
+        "isDataCenterReady": true
```

Comment:
isDataCenterReady is an internal detail that shouldn't be included in the public api

Reply:
Removed the data center instance view entirely.